metabase/metabase
# Add basic model detail page

Epic #27581, milestone 1

Adds a basic model detail page showing model info, making it possible to manage the model, showing its schema and questions based on it. This PR doesn't include any UI to reach the page, it's going to be done in #27628.

### To Verify

1. For any existing model, open the `/model/:id/detail` URL
2. Ensure you can rename the model by clicking on its title
3. Ensure you can change the model description from the right panel
4. Ensure there are links to the chill mode (Explore), query editor (Edit definition), and metadata editor (Edit metadata, under the Schema tab)
5. Ensure you can see questions based on a model under the "Used By" section
6. Ensure you can see model schema under the "Schema" section
7. Ensure the model creator is shown in the right panel
8. Ensure the backing table is shown in the right panel (a table a model was started from, isn't available for native models)
9. Ensure related tables are shown in the right panel (tables linked with a backing table via foreign keys)

### Demo


https://user-images.githubusercontent.com/17258145/211818461-ed8b4db2-40d5-4852-9ec8-d0136acb0af0.mp4


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27311)
